transaction/core/Cargo.toml

```diff
@@ -56,6 +56,7 @@ proptest = { version = "1.0", default-features = false, features = ["default-cod
 rand = "0.8"
 tempdir = "0.3"
 
+mc-blockchain-types = { path = "../../blockchain/types" }
```

Comment:
it looks to me that this dependency is unused, maybe we can remove it?

Reply:
oh i see, it is a dev-dependency.

this is still not a good thing, this caused us to require an "adapt-hack" trick in the past which made test code hard to use.

maybe we can move the `tests/` integration tests to `mc-blockchain-types` ?